### DISCUSSION TITLE
Seat warmer doesn't come on, everything else does - EU

### DISCUSSION THREAD

**Discussion by ghost** [Hyundai-Kia-Connect/hyundai_kia_connect_api#450] [Q&A]
Hi everyone,

I'm using fuatakgun's HA repo (v2.16.2) that uses this library and everything works brilliantly, but can't get any of the seats do anything on any setting/level essentially. Tried various heating / cooling settings, none of them work, but everything else does. LEDs won't come on on the rear armrests, you can't feel anything sitting on any of the front seats either. I also tried cooling because that's easier to detect, but that won't come on either on the front seats that do support cooling. I tried removing the duration parameter as well to see if it gets it working, but no joy. The API call doesn't fail, it gets it all up and running, including steering wheel heating - except anything related to any of the seats.

It's a MY23 I5 made in Nov 22. Is there any software version I need to check on the car to verify if these parameters from the API should be respected? Any other advice please?

Thank you.

**Comment by cdnninja** [comment]
To confirm this works in your native mobile app?

**Reply by ghost** [reply]
Thanks for your response. I got HA installed on a Raspberry Pi4 and I got a button card sending a service call to Bluelink to start air conditioning. I can press that button from either the web browser or a mobile app that can connect to HA on the Raspberry Pi4. Whichever way I press the button doesn't make a difference to this.

**Reply by cdnninja** [reply]
I mean in the Kia Hyundai app.

**Reply by ghost** [reply]
Ah, no. KIA/Hyundai won't let you control seat heating from their apps, that's why someone recommended me using HA/HACS/Bluelink plugin as they say the APIs do have this ability, but the official apps don't. (we had an e-Niro, now have an I5 and I know EV6 ppl waiting for the same functionality - I'm personally waiting since 2020 - the e-Niro times). I presume if this library and the HACS Bluelink plugin deals with these seat warmer params that's because it's possible to do on these cars? (I5/EV6/etc...)

**Reply by cdnninja** [reply]
Yes we only support what the car and native app support. Never more. We can't tell though up front so they are all exposed.

**Reply by ghost** [reply]
Well, I believed the person who said it was working on their I5 as the I5/I6 are the newest/most modern cars that Hyundai makes. Can't imagine what models are these params for if not these. Anyways, thanks for your time.

**Reply by cdnninja** [reply]
What region are you in?

**Reply by ghost** [reply]
I'm in the EU, thank you.
